### PR TITLE
Upgrade ds-pause

### PR DESCRIPTION
**Pull request by d-xo**
- Upgrade `ds-pause` to the latest version
- Integrate the breaking changes
- Minor naming / formatting improvements

### CODE REVIEW

File: src/govActions.sol [hunk 1]
```diff
@@ -13,7 +13,7 @@ contract Setter {
     function init(bytes32) public;
 }
 
-contract Plan {
+contract GovActions {
```
Comment (asymmetric) [inline src/govActions.sol:21]: `Book`?

**Comment by gbalabasquer** [comment]
@xwvvvvwx in order to merge this PR and integrate it to the testchain scripts I'll need to have some function to modify the delay. Are we going to add it to the `pause` or is there any reason to not add that?

File: src/govActions.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/govActions.sol:21]: I agree `Plan` is confusing, I think `Book` as well. They are actions for a proxy.
Not sure if there is a proper 4 letter meme for it.

**Comment by gbalabasquer** [comment]
@xwvvvvwx any comment on this? If we can not add the `setDelay` function, we will need to review the whole process. So the `pause` should be deployed directly in the scripts and not here.

**Comment by asymmetric** [comment]
@gbalabasquer apart from having to deploy the pauses from the scripts, are there actual blockers if the `delay` can't be changed?

I think the issue with changing the `delay` is that we don't know if there are scheduled plans whose semantics change when the `delay` is changed.

For the purposes of a running system, in which we want to change the `delay` of a pause, it seems like it would be better to deploy a new one, and make the existing one read-only (i.e. no new plans can be plotted).

**Comment by gbalabasquer** [comment]
I guess it is possible, but yeah as I said, we should remove the pause integration from `dss-deploy` and put something in the middle (like the ds-proxy used to be), then adding a pause and doing all the set up in the scripts which there should be in another contract/spell (if we want to apply all the auth changes atomically).

> I think the issue with changing the `delay` is that we don't know if there are scheduled plans whose semantics change when the `delay` is changed.

Btw this is not the case. The `eta` is defined in `plot`, so a change in the delay of the pause doesn't matter for those already in the pipeline.

**Comment by asymmetric** [comment]
Oh, you’re totally right. Then I don’t think I see other reasons not to support changing the `delay` via a plan.

**Comment by d-xo** [comment]
Hey @gbalabasquer apologies this slipped off my radar. Will add `setDelay` directly.

**Comment by gbalabasquer** [comment]
Thanks @xwvvvvwx !!!

**Comment by d-xo** [comment]
PR: https://github.com/dapphub/ds-pause/pull/24

**Pull request merged**